source/fuzz/fuzzer_pass_swap_commutable_operands.cpp

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) 2020 Google LLC
```

Comment:
For any files that you contribute from scratch, please use:

// Copyright (c) 2020 [your name]

Reply:
Done.